src/vos/tests/vts_pool.c

```diff
@@ -497,6 +490,36 @@ pool_open_excl_test(void **state)
 	assert_rc_equal(ret, 0);
 }
 
+static void
+pool_interop_create_old(void **state)
+{
+	struct vp_test_args	*arg = *state;
+	uuid_t			uuid;
+	daos_handle_t		poh;
+	int			ret;
+
+	FAULT_INJECTION_REQUIRED();
+
+	uuid_generate(uuid);
+
+	/* DF version too old. */
+	ret = vos_pool_create(arg->fname[0], uuid, VPOOL_256M, 0, 0, 1 /* version */, NULL);
+	assert_rc_equal(ret, -DER_INVAL);
+
+	/* DF version old but supported. */
+	ret = vos_pool_create(arg->fname[0], uuid, VPOOL_256M, 0, 0, VOS_POOL_DF_2_4, NULL);
```

Comment:
No change requested. I suppose a future concern would be changing this line of code to specify whatever is the lowest supported durable format version value, for example after VOS_POOL_DF_2_4 might no longer be supported by future daos software versions.

There is a POOL_DF_VER_1 defined in vos_layout.h - could it be used here to specify as the version argument (referring always to the lowest supported version value)?

Reply:
@kccain, I thought about this and didn't bother to export `POOL_DF_VER_1` to the test, for when `vos` drops `VOS_POOL_DF_2_4` this test will fail to compile (or fail to pass). I can look into exporting `POOL_DF_VER_1` in a separate PR.